<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu_odczytu_i_zapisu</class>
 <widget class="QDockWidget" name="menu_odczytu_i_zapisu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>154</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>154</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>154</width>
    <height>330</height>
   </size>
  </property>
  <property name="floating">
   <bool>true</bool>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetClosable</set>
  </property>
  <property name="allowedAreas">
   <set>Qt::NoDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Zapisopodobne</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sciezka">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyklikaj jak plebs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Wczytaj">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wczytaj</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wpisz ścieżke:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Zapisz">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Zapisz</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_rozszerzenie">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Rozszerzenie</string>
    </property>
    <widget class="QRadioButton" name="radioButton_txt">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>13</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>.txt</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_bin">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>.bin</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_jpg">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>48</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>.jpg</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_format">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>121</width>
      <height>65</height>
     </rect>
    </property>
    <property name="title">
     <string>Format obrazu</string>
    </property>
    <widget class="QRadioButton" name="radioButton_rgb">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>13</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>RGB</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_szary">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Skala szarosci</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_hsv">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>45</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>HSV</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_komunikat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>293</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> </string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
